src/pke/lib/scheme/bfvrns/bfvrns-cryptoparameters.cpp

```diff
@@ -261,7 +271,7 @@ void CryptoParametersBFVRNS::PrecomputeCRTTables(
       }
       m_paramsQlRl[0] =
           std::make_shared<ILDCRTParams<BigInteger>>(2 * n, moduliQR, rootsQR);
-    } else if (multTech == HPSPOVERQLEVELED) {
+    } else if (multTech == HPSPOVERQLEVELED || multTech == HPSPOVERQ) {
```

Comment:
Should we have "else" for this if statement? for multTech BEHZ or invalid. The same question applies to some code in this file below.

Reply:
No, we don't need any of these precomputations for BEHZ. There is separate code in the function for the BEHZ precomputations, but they all have different names.